src/sentry/ingest/transaction_clusterer/datasource/redis.py

```diff
@@ -121,3 +116,31 @@ def _should_store_transaction_name(event_data: Mapping[str, Any]) -> bool:
         return False
 
     return True
+
+
+def _bump_rule_lifetime(project: Project, event_data: Mapping[str, Any]) -> None:
+    from sentry.ingest.transaction_clusterer import rules as clusterer_rules
+
+    applied_rules = event_data.get("_meta", {}).get("transaction", {}).get("", {}).get("rem", {})
```

Comment:
I don't really like this but I'm not sure if there's a better alternative.

Reply:
Looks fine to me!